### PR TITLE
Adds and configures `eslint` and `prettier` for UI 

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/index.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```
Comment (sthaha) [inline ui/src/index.tsx:11]: is this `,` required by eslint?

File: ui/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": ["react", "@typescript-eslint", "prettier"],
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jest": true
+  },
+  "rules": {},
```
Comment (pratap0007) [inline ui/.eslintrc.json:15]: can we add some rule eg. `no-console` or we will add latter ?

File: ui/src/index.tsx (hunk 1 above)
Comment (PuneetPunamiya) [reply ui/src/index.tsx:11]: This was the prettier issue, I have changed the config of prettier :+1:

**Review by sthaha — APPROVED** [review]
/lgtm
/approve 
/meow

**Pull request merged**